### DISCUSSION TITLE
使用FloatingBubble组件导致打包之后的px转换出现问题

### DISCUSSION THREAD

**Discussion by haishengJiao** [youzan/vant#12100] [General]
"vant": "4.6.2",
 "vue": "^3.2.38",
 "postcss-pxtorem": "^6.0.0",
 vite v3.1.0

1.在使用FloatingBubble组件之后，项目打包导致部分px转换不生效
2.去掉FloatingBubble组件之后项目打包px转换没有问题

![Snipaste_2023-07-18_14-06-50](https://github.com/youzan/vant/assets/100760191/c982edb4-268d-4f37-8fb4-d917df363423)
![Snipaste_2023-07-18_14-07-52](https://github.com/youzan/vant/assets/100760191/21b6eb9e-866e-4bab-952b-26488e1492ef)

**Comment by zhousg** [comment]
可以提供下复现的链接
